blocks/library/index.js

```diff
@@ -1,3 +1,4 @@
+import './shortcode';
```

Comment:
Can you explain this change? It makes me nervous that we depend on load order of blocks.

Reply:
Yes, I agree. See https://github.com/WordPress/gutenberg/pull/3326#issuecomment-343150527. My thinking is that we can get away with it for now in `library`, and we could soon start to think of a mechanism based on explicit priority.